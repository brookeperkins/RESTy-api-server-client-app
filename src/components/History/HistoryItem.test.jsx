import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HistoryItem from './HistoryItem';
/* eslint-disable max-len */

describe('HistoryItem component', () => {
  afterEach(() => cleanup());
  it('renders the HistoryItem component', () => {
    const { asFragment } = render(
      <HistoryItem
        url="https://hey-arnold-api.herokuapp.com/api/v1/characters/5da237699734fdcb7bef8f51"
        method="GET"
        body="Here is the body"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
