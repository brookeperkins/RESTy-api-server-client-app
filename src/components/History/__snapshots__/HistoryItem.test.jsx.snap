// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryItem component renders the HistoryItem component 1`] = `
<DocumentFragment>
  <section>
    <h2>
      GET
    </h2>
    <p>
      https://hey-arnold-api.herokuapp.com/api/v1/characters/5da237699734fdcb7bef8f51
    </p>
    <p>
      Here is the body
    </p>
  </section>
</DocumentFragment>
`;
